lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -2337,13 +2337,13 @@ namespace Js
             return si;
     }
 
-#ifdef ENABLE_GLOBALIZATION
     void ScriptContext::UpdateTimeZoneInfo()
     {
         GetTimeZoneInformation(&timeZoneInfo);
+#if !defined(__cplusplus_winrt) && defined(_WIN32)
```

Comment:
Why adding `!defined(__cplusplus_winrt)`? It worked before without it.

Linux has `tzset()`. Does it work here?